public/css/signin_style.css

```diff
@@ -0,0 +1,38 @@
+body {
+    background-image: url("../images/signin_background.jpg");
```

Comment:
You don't need to do relative directory path, since the "public" folder has been set up as the express static directory.